### PR TITLE
MQE-1711: Switch between Developer mode and Production mode takes lon…

### PR DESCRIPTION
**Pull request by soumyau**
…g time and the test end up time out.

### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: dev/tests/verification/TestModule/Test/BasicFunctionalTest.xml [hunk 1]
```diff
@@ -76,6 +76,7 @@
         <grabTextFrom selector=".functionalTestSelector" stepKey="grabTextFromKey1" />
         <grabValueFrom selector=".functionalTestSelector" stepKey="grabValueFromKey1" />
         <magentoCLI command="maintenance:enable" arguments="&quot;stuffHere&quot;" stepKey="magentoCli1"/>
+        <magentoCLI command="maintenance:enable" arguments="&quot;stuffHere&quot;" timeout="120" stepKey="magentoCli2"/>
```
Comment (tomreece) [inline dev/tests/verification/TestModule/Test/BasicFunctionalTest.xml]: Can we add two `<magentoCLISecret ..>` actions to this verification test? Use default and 120 timeout like magentoCLI.

File: docs/test/actions.md [hunk 2]
```diff
@@ -1262,6 +1262,7 @@ Attribute|Type|Use|Description
 ---|---|---|---
 `command`|string |optional| CLI command to be executed in Magento environment.
 `arguments`|string |optional| Unescaped arguments to be passed in with the CLI command.
+`timeout`|string|optional| Number of seconds CLI command can run without outputting anything.
```
Comment (tomreece) [inline docs/test/actions.md]: Can we add `magentoCLISecret` to the documentation too?

File: dev/tests/verification/TestModule/Test/BasicFunctionalTest.xml (hunk 1 above)
Comment (soumyau) [reply dev/tests/verification/TestModule/Test/BasicFunctionalTest.xml]: Added the steps to this verification test.

File: docs/test/actions.md (hunk 2 above)
Comment (soumyau) [reply docs/test/actions.md]: `magentoCLISecret` is not an exposed action, it's internally mapped to magentoCLI based on references to _CREDS; may not need documentation imo.

Comment (tomreece) [reply docs/test/actions.md]: Makes sense to me. No action needed.

**Pull request merged**